<launch>

  <node pkg="amigo_arm_navigation" type="joint_trajectory_publisher" respawn="false" name="r_joint_trajectory_publisher" output="screen">
  <remap from = "controller_command" to = "arm_right_controller/controller_command"/>
  <remap from = "joint_coordinates" to = "arm_right_controller/joint_coordinates"/>
  </node>
  
</launch>
